docker-compose.yml

```diff
@@ -16,24 +16,19 @@ mysql:
     MYSQL_USER: teamcity
     MYSQL_PASSWORD: teamcity
     MYSQL_ALLOW_EMPTY_PASSWORD: yes
-agentone:
+agent:
```

Comment:
I had three predefined agents because there is no default `scale` option when you start it for the first time.